heka/files/toml/filter/lma_alarm.lua

```diff
@@ -0,0 +1,47 @@
+local M = {}
+setfenv(1, M) -- Remove external access to contain everything in the module
+
+local alarms = {
+{%- for _trigger_name in alarm.triggers %}
+{%- set _trigger = trigger.get(_trigger_name) %}
+{%- if _trigger is not none and _trigger.get('enabled', True) %}
```

Comment:
Here I am using Jinja2's `none` built-in test (see http://jinja.pocoo.org/docs/dev/templates/#none). The same way you do `{% if foo is defined %}`. `defined` and `none` are both built-in tests.